content/guides/farmlands-pos-integration.md

```diff
@@ -2,6 +2,10 @@
 title: Farmlands POS Integration Guide
 img: /farmlands-pos-integration-cover.jpg
 description: Centrapay and Farmlands have entered into a partnership to allow Farmlands Card Partners to accept Farmlands Card payments at the point of sale.
+nav:
+  path: /connections/farmlands/farmlands-pos-integration
```

Comment:
Shouldn't this path be `/connections/farmlands/pos-integration-guide`

Reply:
Hmm the last part of the nav path actually doesn't matter. This is because once we get to the last part, and we see that there are no more children, we set the title of the navigation menu to the `title` or `nav.title` in the front matter.

I will rework the solution so that we don't need to define that last part i.e. `/connections/farmlands`. I think this makes more sense as it is saying this page belongs to the `Connections` section under the `Farmlands` sub-section.